gulpfile.ts

```diff
@@ -79,7 +79,10 @@ export function copyJs() {
     "./node_modules/jsvectormap/dist/maps/world-merc.js",
     "./node_modules/select2/dist/js/select2.min.js",
     "./node_modules/jsbarcode/dist/JsBarcode.all.min.js",
-  ]).
+  ], {
```

Comment:
We need to add ""./node_modules/jquery/dist/jquery.js"" in the list above, otherwise it's not copied and product opener does not find jquery

Reply:
Ah, sorry!  Yes, I had that problem once or twice during local testing too.  I committed a fix but must have dropped it during a rebase.  I'll push that fix in a few moments.

Reply:
Ok, d606852fc30e2f3083ced004b0c7b265f5ecd257 applies that change.

Reply:
Note: in a [previous comment](https://github.com/openfoodfacts/openfoodfacts-server/pull/11109#issuecomment-2532716651) I reported test results from `jquery` v3.7.1 -- a much more recent and preferable version if we can use it.  But the commit ID I mentioned (fd7fb7fc840ffe94b365a7d17cc4668d749b4d32) also lacks this `gulpfile.ts` copy-list fix.

Reply:
(since yesterday, I've reconfirmed with the fix in place that indeed, v2.2.4 is OK - but v3.7.1 is not)